src/providers/ibmcloud/mod.rs

```diff
@@ -112,6 +126,29 @@ impl IBMGen2Provider {
         }
         output
     }
+
+    /// Find the SSH keys in the vendordata file
+    fn fetch_ssh_keys(vendordata_vec: Vec<u8>) -> Result<Vec<String>> {
+        // Parse MIME format from vendor-data file
+        let vendor_data_mail = parse_mail(&vendordata_vec).unwrap();
```

Comment:
This `unwrap` should be replaced by a `.chain_err(...)?` or `map_err(...)?` instead.

Reply:
Changed in the next commit.